scss/components/_big-date.scss

```diff
@@ -0,0 +1,74 @@
+/** Big calendar type date styling **/
+.civicase__big-date {
```

Comment:
This is not a new component. Instead its a variant of `civicase__activity-date`. So should be part of same file. That is `_activity-card.scss` or `_activity-card--big.scss`.